obspy/core/stream.py

```diff
@@ -3118,6 +3119,78 @@ def remove_sensitivity(self, *args, **kwargs):
             tr.remove_sensitivity(*args, **kwargs)
         return self
 
+    def stack(self, group='all', type='normal', npts_tol=0):
+        """
+        Return stream with traces stacked by the same selected metadata.
+
+        :param group: Stack waveforms together which have the same metadata given
+            by this parameter. The parameter should name the corresponding keys
+            of the stats object, e.g. `'{network}.{station}'` for stacking all
+            locations and channels of the stations and returning a stream
+            consisting of one stacked trace for each station. This parameter can
+            take two special values, `'seedid'` which stacks the waveforms by
+            seedid and `'all'` (default) which stacks together
+            all traces in the stream.
+        :param type: Type of stack, one of the following:
+            `'normal'`: normal stack (default),
```

Comment:
The 'normal' stack is normal, is a bit of a tautology.

Reply:
'normal' should be 'linear' or 'average', maybe?